src/sql/workbench/browser/designer/interfaces.ts

```diff
@@ -187,6 +187,14 @@ export interface DesignerTableProperties extends ComponentProperties {
 	 * Whether user can remove rows from the table. The default value is true.
 	 */
 	canRemoveRows?: boolean;
+	/**
+	 * Whether user can move rows from one index to another. The default value is true.
+	 */
+	canMoveRows?: boolean;
+	/**
+	 * Whether user can insert rows at a given index to the table. The default value is true.
+	 */
+	canInsertRows?: boolean;
```

Comment:
default value for both `canMoveRows` and `canInsertRows` should be false
https://github.com/microsoft/sqltoolsservice/blob/99d0959291971a22552b58425a703109acf3f45e/src/Microsoft.SqlTools.ServiceLayer/TableDesigner/Contracts/Components/TableProperties.cs#L52